extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -325,15 +326,19 @@ describes.realWin(
 
       it('should expand the promptUISrc', async () => {
         const config = dict({
-          'promptUISrc': 'https://example.test/?cid=CLIENT_ID&r=RANDOM',
+          'promptUISrc':
+            'https://example.test/?cid=CLIENT_ID&r=RANDOM&clientconfig=CONSENT_INFO(clientConfig)&cpid=CONSENT_PAGE_VIEW_ID_64',
```

Comment:
This line got unwieldy. 

1. Let's split by query parameter. This should make the diff more stable in the future.
2. Let's move `RANDOM` to the end like it was initially. (`RANDOM` is unlike the others in that it's not allowed).

```suggestion
            'https://example.test/?' +
            'cid=CLIENT_ID&' +
            'clientconfig=CONSENT_INFO(clientConfig)&' +
            'cpid=CONSENT_PAGE_VIEW_ID_64&' +
            'r=RANDOM',
```